selfdrive/thermald.py

```diff
@@ -135,6 +137,36 @@ def handle_fan_uno(max_cpu_temp, bat_temp, fan_speed, ignition):
 
   return new_speed
 
+def get_network_type():
+  wifi_check = android.parse_service_call_string(android.service_call(["connectivity", "2"]))
+  cell_check = android.parse_service_call_unpack(android.service_call(['phone', '59']), ">q")
+  cell_networks = {
```

Comment:
This looks very android specific. Maybe move the service call + parsing into the android library?